.github/lsan.supp

```diff
@@ -0,0 +1 @@
+leak:libCore.so
```

Comment:
```suggestion
leak:libCore.so
leak:libedm4hep.so
```

Reply:
Would that not hide EDM4hep objects allocated by EICrecon?

Reply:
That's what I was wondering (and reason why I hadn't committed it after making the comment). Needs testing but my local EICrecon copy is tied up...

Reply:
Which leak are you looking to suppress by filtering out EDM4hep?

Reply:
The numerous occurrences of e.g. https://github.com/eic/EICrecon/actions/runs/5448228500/jobs/9911415555?pr=722#step:5:4037:
```
Indirect leak of 336 byte(s) in 14 object(s) allocated from:
    #0 0x7fa81439d4c8 in operator new(unsigned long) ../../../../src/libsanitizer/asan/asan_new_delete.cpp:95
    #1 0x7fa80f567c2b in edm4hep::(anonymous namespace)::createBuffers(bool) [clone .lto_priv.2] (/usr/local/lib/libedm4hep.so+0x65c2b)
```

But I was going to test what else it hides.

Reply:
How about 
```suggestion
leak:libCore.so
leak:createBuffers
```
then?

Reply:
Or `leak:edm4hep::*::createBuffers` to make sure it's still in EDM4hep. I'll give that a try.

Reply:
```suggestion
leak:libCore.so
leak:edm4hep::*::createBuffers
```